fix: deleted the empty main.css and remove all import references bug:…

Idris01 opened the pull request:
This PR is 
- Using flexBox to place element in the page
- Use images and backgrounds to enhance the look of the website.

Idris01 committed:
fix: deleted the empty main.css and remove all import references bug: cleared all the bugs using npm dependencies

Idris01 committed:
feat: Add svg images for the background and menu icons
, Adds mobile.css file to take care of the milestone mobile requirements
docs: Update the README by adding correct author twitter name
style: Add new styles to theindex and reset
feat: update the app_screenshot.png picture to match the recent view

Idris01 committed:
feat: Add rectangel.svg for background
style: Update the mobile style to meet screen size

Idris01 committed:
style: Include rectangle.svg in background styles

Idris01 committed:
docs: Add live demo link

MrTaaJ commented on README.md:
```diff
@@ -1,4 +1,4 @@
-# Professional Portfolio 
+# Professional Portfolio
 
 > This is my Professional Portfolio check it out to get my details.
```
I think this introductory part of the README can be updated to a more standard format. Although the project is still under development, I will suggest using a third-person pronoun; like "This is a Professional Portfolio by ${name}; ${a quick guide of what it does}. ${Call to action, which I presume this is doing already - 'check it out to get my details.'}".

Idris01 replied:
Thank you for the update, I have reformed it as suggested

MrTaaJ commented on index.html:
```diff
@@ -4,7 +4,128 @@
+    <header>
+      <h2 class="brand">
+        RefiDev
+      </h2>
```
Since this is a logo, it is best if it is inside an a-tag.
`<h2 class="brand"> <a class="" href=""> RefiDev </a></h2>`

Idris01 replied:
suggestions implemented

MrTaaJ commented on index.html:
```diff
@@ -4,7 +4,128 @@
+          <a href="" class="nav-link">Contact</a>
+        </li>
+      </ul>
+      <label for="hidden-toggler" class="nav-button"> </label>
```
Since this is a nav button, it is best for it to be in a nav-tag.

Idris01 replied:
your suggestion is actually one of the best practices, thanks

MrTaaJ commented on index.html:
```diff
@@ -4,7 +4,128 @@
+                  fill="#505F79"
+                />
+              </svg>
+            </a>
```
It is best to put this in a ul-li list. The anchor tag will be inside the li list.
`<ul><li><a><svg></svg></a></li><li><a><svg></svg></a></li></ul>`

Idris01 replied:
An adjustment had been made to make the images more compact

MrTaaJ commented on css/index.css:
```diff
@@ -1,4 +1,39 @@
+    49% 4%,
+    47% 5%,
+    50% 84%;
+}
```
This is very good, but try to adjust the positions; some of the background images are not well aligned with the text. Alternatively, you can download the background file as png from Figma. Please follow the suggested height for the main section as it is on Figma.
![header-shapes mobile-refidas](https://user-images.githubusercontent.com/35946662/137947650-3c23958d-8342-4394-84a5-1709a3595b66.png)

Idris01 replied:
the background image has been updated

MrTaaJ commented on css/mobile.css:
```diff
@@ -0,0 +1,133 @@
+}
+
+.connect {
+  display: flex;
```
We can try to put some repetitive codes into a class and add the class in the HTML tag.
Example
.d-flex{display:flex}

Then add a class of d-flex to any section that needs to be flexed.

Idris01 replied:
This suggestion is helpful as it aids overall code readability

MrTaaJ reviewed (commented):
This is very good and well written. It only requires little changes.

Idris01 committed:
doc: Refactor the introduction in the README file

The introductory section of the README is restructured by  using third person references  to define what the project is about

Idris01 committed:
feat: Add background and social icons svg images

Idris01 committed:
refactor: Remove svg images and from index file, Add navigation tag to navigation items and add corresponding custom classes to other tags

Idris01 committed:
style: Include custom classes in the reset file,
update index.css with needed contents from the mobile.css file,
and delete the mobile.css file

Idris01 committed:
refactor: Remove dupliate nav in the index.css,Runs linting rests

MrTaaJ commented on index.html:
```diff
@@ -4,7 +4,70 @@
+    </header>
+    <main>
+      <section class="headlines">
+        <div id="background">
+          <img src="./img/background_mobile.svg" alt="" />
+        </div>
```
I do not mean that the image should be added in html and used as background by setting the position. Perhaps you can try using it as the background image for the main tag or .headlines.

`main {
  height: 590px;
  background-color: #1c1a19;
  background-image: url(/img/mobile-background.png);
  background-position: bottom -40px right 0;
  background-repeat: no-repeat;
}`

Something like this should do the trick.

Idris01 replied:
background adjusted

MrTaaJ commented on img/menu.svg:
```diff
@@ -0,0 +1,42 @@
+	<path d="M0,3.875c0-1.104,0.896-2,2-2h20.75c1.104,0,2,0.896,2,2s-0.896,2-2,2H2C0.896,5.875,0,4.979,0,3.875z M22.75,10.375H2
+		c-1.104,0-2,0.896-2,2c0,1.104,0.896,2,2,2h20.75c1.104,0,2-0.896,2-2C24.75,11.271,23.855,10.375,22.75,10.375z M22.75,18.875H2
+		c-1.104,0-2,0.896-2,2s0.896,2,2,2h20.75c1.104,0,2-0.896,2-2S23.855,18.875,22.75,18.875z"/>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
```
Did you take a look at this as suggested in the video?

Idris01 replied:
reviewed and implemented

MrTaaJ commented on index.html:
```diff
@@ -4,7 +4,70 @@
+        <div class="connect">
+          <h4 class="connect-heading">let's connect</h4>
+          <div class="connect-icons">
+            <a href="#" class="twitter">
+              <img src="./img/twitter.svg" alt="" />
+            </a>
+
+            <a href="https://www.linkedin.com/in/idris-adebowale-ab4208ab/" class="linkedin">
+              <img src="./img/linkedin.svg" alt="" />
+            </a>
+
+            <a href="#" class="myscpace">
+              <img src="./img/myspace.svg" alt="" />
+            </a>
+            <a href="#" class="pintrest">
+              <img src="./img/pintrest.svg" alt="" />
+            </a>
+            <a href="https://github.com/idris01" class="">
+              <img src="./img/github.svg" alt="" />
+            </a>
```
It is best to put this in a ul-li list. The anchor tag will be inside the li list.
`<ul><li><a><svg></svg></a></li><li><a><svg></svg></a></li></ul>`

I guess you miss this part of putting it in a list item.

Have you heard of ion-icon or boxicon? We can use this method for icons instead of img.
Please do not get me wrong, using an image is also acceptable, but it may increase the file size unlike using ion-icon or boxicon that will only be added like a link to their respective website. 
If you are comfortable with the img, it's fine. Don't forget to add them in a list, though.

Idris01 replied:
noted and adjusted

Idris01 replied:
All suggestions considered and implemented

MrTaaJ reviewed (commented):
This is okay. You can check the comments and go ahead with merging when you have been approved to merge.

Idris01 committed:
fix: Add connect icons inside ul for proper representation,
style: Add background image to  the .headlines

Idris01 commented:
The suggestions made on the subsequent reviews have been checked and implemented, Thank you
